src/Controls/samples/Controls.Sample.WinUI/Maui.Controls.Sample.WinUI.csproj

```diff
@@ -21,4 +24,7 @@
     <ProjectReference Include="..\..\src\Core\Controls.Core-net6.csproj" />
     <ProjectReference Include="..\Controls.Sample\Maui.Controls.Sample-net6.csproj" />
   </ItemGroup>
+
+  <!-- Normally this import would be automatic through the NuGet package. We do it manually for projects inside this solution. -->
+  <Import Project="..\..\..\BlazorWebView\src\core\build\net6.0-windows\Microsoft.AspNetCore.Components.WebView.Maui.targets" Condition="$(TargetFramework.StartsWith('net6.0-windows'))" /> 
```

Comment:
Rather than adding these in different places manually, would it work to add `Directory.Build.props` and `Directory.Build.targets` files?

Also is there any symbol that could be used in place of the `net6.0-windows`? We know for sure that version number is going to get stale.